packages/eventual-send/src/types.d.ts

```diff
@@ -0,0 +1 @@
+export type Callable = (...args: any[]) => any;
```

Comment:
`💬 I'm so lonely out here` said the type which is only being referenced within `E.js` at this point!

@michaelfig move to `E.js`?

Reply:
You can put it where you'd like, but please change to:
```suggestion
export type Callable = (...args: unknown[]) => any;
```